config/locales/en.yml

```
@@ -296,6 +296,7 @@ en:
     require_mfa_enabled: Your multifactor authentication has not been enabled. You have to enable it first.
     setup_recommended: For protection of your account and your gems, we encourage you to set up multifactor authentication. Your account will be required to have MFA enabled in the future.
     strong_mfa_level_recommended: For protection of your account and your gems, we encourage you to change your MFA level to "UI and gem signin" or "UI and API". Your account will be required to have MFA enabled on one of these levels in the future.
+    ui_only_warning: UI Only is being removed; please choose another MFA level
```

Perhaps deprecated would be a better term to use instead of removed. I would also suggest which levels to change to.

```suggestion
    ui_only_warning: UI Only is deprecated. We encourage you to change your MFA level to "UI and gem signin" or "UI and API".
```

---

Can you please elaborate why deprecation makes more sense? Deprecation means that while the feature is still supported, it will be removed in future. By feature here we mean ability to update MFA level to ui_only. 
I would recommend not using "encourage" as well when the user doesn't have any option beside using non ui_only levels.

---

> Deprecation means that while the feature is still supported, it will be removed in future. 

I thought that deprecated would be the correct choice as it is still supported for those on the level currently. While I'm typing this, I see why that nuance would not affect the current user. My bad,  👍 to change it back to removed.

> "encourage" as well when the user doesn't have any option beside using non ui_only levels.

The user has the option to select the disabled level. However, I'm for having more strict language like `please`  in the original.

---

Thank you for clarifying. We can use something like.. "Updating MFA level to ui_only is not supported. Please use..".